content/plugins/aws.md

```diff
@@ -46,12 +53,30 @@ All cloud resource nodes have common properties:
 **Properties**
 
   * `use_external_resource` a boolean for setting whether to create the resource or use an existing one. See the [using existing resources section](#using-existing-resources). Defaults to `false`.
-  * `resource_id` this can be the name of a resource when one is supported (security group and keypair) or the ID of an existing resource when the `use_external_resource` property is set to `true` (see the [using existing resources section](#using-existing-resources)). Defaults to `''` (empty string).
+  * `resource_id` The ID of an existing resource when the `use_external_resource` property is set to `true` (see the [using existing resources section](#using-existing-resources)). Defaults to `''` (empty string). When using a new (not external) security group or key pair, this will become the resource name.
   * `aws_config` a dictionary that contains values you would like to pass to the connection client. For information on values that are accepted, please see [boto documentation](http://boto.readthedocs.org/en/latest/ref/ec2.html#boto.ec2.connection.EC2Connection)
 
+Every time you manage a resource with Cloudify, we create one or more clients with AWS API. You specify the configuration for these clients using the aws_config property. It should be a dictionary, with the following values:
```

Comment:
this is a duplication of the bullet point above?
the bullet point above says "for more info see there" but a line after that you explain it? I suggest re-writing this section so it will be easier to follow